includes/apple-exporter/components/class-slug.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Publish to Apple News: \Apple_Exporter\Components\Slug class
+ *
+ * @package Apple_Exporter
+ */
+
+/**
+ * A slug.
+ *
+ * @since 0.2.0
+ */
+namespace Apple_Exporter\Components;
+
+class Slug extends Component {
+
+	/**
+	 * Get all specs used by this component.
+	 *
+	 * @return array
+	 * @access public
+	 */
+	public function get_specs() {
+		return $this->specs;
+	}
+
+	/**
+	 * Get a spec to use for creating component JSON.
+	 *
+	 * @since 1.2.4
+	 * @param string $spec_name The name of the spec to fetch.
+	 * @access protected
+	 * @return array The spec definition.
+	 */
+	protected function get_spec( $spec_name ) {
+		if ( ! isset( $this->specs[ $spec_name ] ) ) {
+			return null;
+		}
+
+		return $this->specs[ $spec_name ];
+	}
+
+	/**
+	 * Register all specs for the component.
+	 *
+	 * @access public
+	 */
+	public function register_specs() {
+		
+		$this->register_spec(
+			'json',
+			__( 'JSON', 'apple-news' ),
+			array(
+				'role' => 'heading',
+				'text' => '#text#',
+			)
+		);
+
+		$this->register_spec(
+			'default-slug',
+			__( 'Style', 'apple-news' ),
+			array(
+				'textAlignment' => '#text_alignment#',
+				'fontName' => '#slug_font#',
+				'fontSize' => '#slug_size#',
+				'lineHeight' => '#slug_line_height#',
+				'tracking' => '#slug_tracking#',
+				'textColor' => '#slug_color#',
```

Comment:
Fix arrow alignment:
```suggestion
				'textAlignment' => '#text_alignment#',
				'fontName'      => '#slug_font#',
				'fontSize'      => '#slug_size#',
				'lineHeight'    => '#slug_line_height#',
				'tracking'      => '#slug_tracking#',
				'textColor'     => '#slug_color#',
```